pkg/didcomm/protocol/introduce/service.go

```diff
@@ -523,7 +542,7 @@ func (s *Service) saveTransitionalPayload(id string, data transitionalPayload) e
 		return fmt.Errorf("marshal transitional payload: %w", err)
 	}
 
-	return s.store.Put(fmt.Sprintf(transitionalPayloadKey, id), src)
+	return s.store.Put(fmt.Sprintf(transitionalPayloadKey, id), src, storage.Tag{Name: transitionalPayloadKey})
```

Comment:
General note... it's possible that many of the tag names I'm using could be shortened. For now, I just used the existing constants defined in these files - some of which may have extraneous info. I didn't want to risk changing more stuff than I had to... (and I already had to change a ton of files).

On a similar note, some of the keys that are being used could probably be simplified - before the long key names were used in order to allow for filtering via the old Iterator method, since that relied on using certain key suffixes/prefixes.

I think this is a task for the future...